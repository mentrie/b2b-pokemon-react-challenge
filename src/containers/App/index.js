import React, { Component } from 'react';
import Footer from '../../components/Footer';
import Header from '../../components/Header';
import Main from '../../containers/Main';

import '../../static/styles/main.css';

class App extends Component {
  render() {
    return (
      <div>
        <Header />
        <Main />
        <Footer />
      </div>
    );
  }
}

export default App;
