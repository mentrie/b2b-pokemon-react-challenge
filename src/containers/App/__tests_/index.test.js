import React from 'react';
import App from '../index';

let component;
beforeAll(() => {
  component = shallow(<App />);
});

test('renders without crashing', () => {
  expect(component).toMatchSnapshot();
});

test('should contain Footer component', () => {
  const component = shallow(<App />);
  expect(component.find('Footer').length).toBe(1);
});

test('should contain Header component', () => {
  const component = shallow(<App />);
  expect(component.find('Header').length).toBe(1);
});
