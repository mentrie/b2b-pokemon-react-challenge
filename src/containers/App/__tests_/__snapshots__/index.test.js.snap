// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <Header />
  <withRouter(Main) />
  <Footer />
</div>
`;
